config.js

```diff
@@ -466,6 +466,15 @@ var config = {
     //    downloadAppsUrl: 'https://docs.example.com/our-apps.html'
     // },
 
+    // Options related to the remote participant menu.
+    // remoteVideoMenu: {
+    //     // If set to true the 'Kick out' button will be disabled.
+    //     disableKick: true
+    // },
+
+    // If set to true all muting operations of remote participants will be disabled.
+    // disableRemoteMute: true,
```

Comment:
Why is this not part of the remoteVideoMenu thing?

Reply:
Because on true, for web, the overflow menu item 'Mute everyone' will be also disabled.

Reply:
Then let'ss have an option for every item we need to disable. As it stands I don't undderstand what this option does.

Reply:
Yes, I see the value in that. But since those options are all inside the "remote video menu" we should have it there.

Reply:
@saghul I'm sorry if I'm missing something but as I said NOT all remote mute buttons are in the remote video menu. There is a 'Mute everyone' button in the secondary overflow menu which is part of the toolbox (see the screenshot). Let me know if I missed something!
![Screenshot 2020-03-29 at 3 08 37 PM](https://user-images.githubusercontent.com/1729265/77859483-55ceb300-71cf-11ea-935e-dcafaac35c45.png)

Reply:
Ah right, sorry my bad. That begs the question of course: why do we need this if there is already a way to do it with INTERFACE_CONFIG?

Reply:
I got really lost now! Sorry!

Why do we need what? Also what do we have in INTERFACE_CONFIG?